ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: ipi-conf-aws-outpost
```

Comment:
If this chain only includes one ref, I guess we do not need to introduce the chain.

Reply:
so just remove *-chain from folder?

to ur questions above:

Reply:
> 'outpost' is an on-premise hardware of Amazon
Sounds like need a specific aws account, right? If right, I think you need start from the very beginning of CLUSTER_PROFILE creation.

Reply:
Why create a new cluster_profile, and not just inject my domain + credentials?  (it is going to be temp solution until 'regular' ci aws account will have it's own 'outpost'.  right?